gnosis_vpn-lib/src/connection/options.rs

```diff
@@ -26,6 +27,13 @@ pub struct BufferSizes {
     pub(super) main: String,
 }
 
+#[derive(Clone, Debug)]
+pub struct MaxSurbUpstream {
+    pub(super) bridge: String,
+    pub(super) ping: String,
+    pub(super) main: String,
+}
```

Comment:
why not have this `pub` globally? these do not see to hold any specific invariant warranting a constructor...

Reply:
agreed